### PR TITLE
update lsp-client to fix lsp0 method name

### PR DESCRIPTION
**Pull request by yanganto**

### CODE REVIEW

File: kld/src/ldk/mod.rs [hunk 1]
```diff
@@ -57,6 +59,7 @@ pub(crate) type LiquidityManager = ldk_lsp_client::LiquidityManager<
     Arc<KeysManager>,
     Arc<KldLogger>,
     Arc<KeysManager>,
+    Arc<DummyFilter>,
```
Comment (JosephGoulden) [inline kld/src/ldk/mod.rs:62]: you can use `Arc<dyn Filter + Send + Sync>` as we just set the filter to None anyway. Then there is no need for the DummyFilter.

**Review by JosephGoulden — CHANGES_REQUESTED** [review]

File: kld/src/ldk/mod.rs (hunk 1 above)
Comment (yanganto) [reply kld/src/ldk/mod.rs]: Good idea.

**Review by JosephGoulden — APPROVED** [review]

**Comment by paulmattei** [comment]
bors merge

**Comment by paulmattei** [comment]
error: path '/nix/store/gvrpk9v6vcq74dxgqd22gzybsy9qbiym-xscreensaver.pam' is not valid ? @yanganto

**Pull request merged**